test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/BookkeeperProperties.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.test.system.framework.services.kubernetes;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * POJO class for bookkeeperProperties.
+ */
+@Getter
+@Setter
```

Comment:
Please use lombok + Jackson annotations to make these immutable
https://www.baeldung.com/jackson-deserialize-immutable-objects
https://projectlombok.org/features/Builder